test/core/surface/public_headers_must_be_c89.c

```diff
@@ -74,5 +74,18 @@
 #include <grpc/support/time.h>
 #include <grpc/support/tls.h>
 #include <grpc/support/useful.h>
+#include <grpc_c/bidi_streaming_blocking_call.h>
```

Comment:
Added a field in `build.yaml` for this purpose. 144391adc6af7d8a2ae49bc60a8109201953a335